<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppWindow</class>
 <widget class="QMainWindow" name="AppWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1276</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius: 12px;
}</string>
   </property>
   <widget class="QListWidget" name="lwToDoList">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>35</y>
      <width>231</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblToDoList">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>0</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ToDo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leInput">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>305</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>35</y>
      <width>821</width>
      <height>581</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>110</number>
    </attribute>
    <row>
     <property name="text">
      <string>00h</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>1h</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2h</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3h</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4h</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5h</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6h</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>7h</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>8h</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>9h</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>10h</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>11h</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>12h</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>13h</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>14h</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>15h</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>16h</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>17h</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>18h</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>19h</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>20h</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>21h</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>22h</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>23h</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Mon</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tue</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Wed</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Thu</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fri</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sat</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sun</string>
     </property>
    </column>
   </widget>
   <widget class="QListWidget" name="lwFriends">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>345</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>91</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout2"/>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>1074</x>
      <y>260</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources.qrc">
      <normaloff>:/resources/images/trash-2.svg</normaloff>:/resources/images/trash-2.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLogout">
    <property name="geometry">
     <rect>
      <x>1197</x>
      <y>361</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources.qrc">
      <normaloff>:/resources/images/user-x.svg</normaloff>:/resources/images/user-x.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>425</y>
      <width>361</width>
      <height>187</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSettings">
    <property name="geometry">
     <rect>
      <x>1133</x>
      <y>361</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources.qrc">
      <normaloff>:/resources/images/settings.svg</normaloff>:/resources/images/settings.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSmartPlan">
    <property name="geometry">
     <rect>
      <x>1197</x>
      <y>35</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources.qrc">
      <normaloff>:/resources/images/calendar.svg</normaloff>:/resources/images/calendar.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1276</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
